spec/models/video_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'spec_helper'
+require 'shoulda/matchers'
```

Comment:
If you include it in the `spec_helper` it will be included for you automatically.